modules/updating-sno.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-between-minor.adoc
+
+= Updating single node {product-title}
+
+You can update, or upgrade, a single-node {product-title} cluster using either the console or CLI methods. 
+
+However, because single node {product-title} does not support high availability, the following prerequisites are not required: 
+
+* having a recent etcd backup
```

Comment:
@hexfusion so you say that we should encourage customers to perform the etcd backup? +1 from me, just that from what you said, the restore is not supported currently. WDYT?

Reply:
@romfreiman Restore should be supported as long as the existing node has not been destroyed. So you have 2 clear failure cases.

- control-plane node exists restore would be supported in my mind.
- control-plane node does not exist. this is basically a fresh install and not supported.

Reply:
@hexfusion @romfreiman @sinnykumari @rioliu-rh So, we should not state that the  _having a recent etcd backup_ prerequisite is not required? 
We would just have:
_However, because single-node OpenShift Container Platform does not support high availability, the prerequisite to pause the `MachineHealthCheck` resources is not required._

Reply:
@hexfusion @romfreiman @sinnykumari @rioliu-rh  ^^

Reply:
makes sense to me I will defer to @romfreiman